src/Context/ContextFactory.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Roave\BetterReflection\Context;
+
+interface ContextFactory
```

Comment:
Initially I created a `Context` interface too, but didn't want to push it too far.